Tuist/Package.swift

```diff
@@ -16,7 +16,7 @@ import PackageDescription
 #endif
 
 let package = Package(
-  name: "MyApp",
+  name: "Gabbangzip",
   dependencies: [
     .package(url: "https://github.com/pointfreeco/swift-composable-architecture.git", from: "1.8.0"),
```

Comment:
P4
```suggestion
    .package(url: "https://github.com/pointfreeco/swift-composable-architecture.git", exact: "1.8.0"),
```
TCA 버전 업데이트에 따른 API 변경에 신경을 덜 쓰도록 버전을 고정하는 것이 빠른 MVP 개발에 더 도움이 될 것 같은에 어떤가요??

Reply:
좋은 생각입니다 👍